### ISSUE TITLE
New Ideas, Suggestions or Critiques

### ISSUE THREAD

**Issue by unshakenme** [unshakenme/OSPAT#18]
To avoid spamming other Issue topics, let's use this Issue to discuss new ideas/suggestions/critiques as they come up.

**Comment by CheekeyMonkey** [comment]
I think I have an interesting idea, but it would need a coder to help.

A big issue with taking psychedelics for the first time is the idea of psychedelic anxiety syndrome i.e. people become quite nervous because they've heard different stories surrounding psychedelics. They start to concentrate on the various risks instead of thinking in relative terms. A lot of scientists like Carhartt-Harris have recently been exploring out of the people who have taken psychedelics what personal characteristics are related to an increased risk of adverse effects. This has led to the best practice publication of what they call _exclusion criteria_ for psychedelic clinical trials. i.e. if you have a certain characteristic, such as schizophrenia, you will be barred from entering in the clinical trials as you would have a high probability of having an adverse effects.

I work in a cooperative that has collected these exclusion criteria together, [see here](https://burnzero.com/Psychedelic_Therapy_Inclusion_and_Exclusion_Criteria). Each exclusion, for instance, Personality Disorder (PD), has an associated, standardised screening questionnaire, for PD it's the McLean Screening Instrument. This is openly available online (https://novopsych.com.au/assessments/diagnosis/mclean-screening-instrument-for-bpd-msi-bpd/).

The idea would be to list all of the exclusions, and corresponding tests, like this: 

- Moderate major depressive disorder - Hamilton Depression Rating Scale: https://www.mdcalc.com/calc/10043/hamilton-depression-rating-scale-ham-d
- Treatment Resistance - Massachusetts General Hospital Antidepressant Treatment Response Questionnaire (MGH-ATRQ) - https://www.nppsychnavigator.com/Clinical-Tools/Psychiatric-Scales/Antidepressant-Treatment-Response-Questionnaire-(A#:~:text=The%20Massachusetts%20General%20Hospital%20(MGH,major%20depressive%20disorder%20(MDD).
- Borderline Personality Disorder - McLean Screening Instrument
- Significant suicide risk - Columbia-Suicide Severity Rating Scale (C-SSRS)

The idea would be someone who is nervous about taking psychedelics would go to a site and see the above 4+ tests and then do them sequentially, at the end of which a report is generated which would give them an overall risk of adverse effects.

**Comment by unshakenme** [comment]
I think this is a great idea! I'm curious why Hamilton Depression Rating Scale vs. the standarized PHQ-9 for depression screening?

I took a look at your website list of exclusion criteria and also have some questions about a few items on the list (namely 2 re: ketamine and 4 re: therapy?). While I would understand how these may be relevant for research purposes (so as not to skew results), I don't understand why they would matter for general integration therapy purposes.

**Comment by CheekeyMonkey** [comment]
Good point I guess the PHQ9 would be better as it's self-administered, Ive added this to the main project page: https://burnzero.com/Psychedelic_Therapy_Risk_Analysis_Tool

The exclusion of ketamine therapy was really to control the end results, i.e. so that the recorded end effect of whatever psychedelic is taken and not potentially the effects of another psychoactive.

As for 4, this is here as, if a patient is undergoing a type of talk therapy which in the opinion of the psych administering is keeping them above self harm, its good to exclude people as you wouldn't want to disturb current, stable therapy.
